web/templates/scheduled_messages_indicator.hbs

```diff
@@ -0,0 +1,12 @@
+
+<div class="scheduled_message_indicator">
+    {{#if several_messages}}
+    {{#tr}}You have <z-link>{scheduled_message_count} scheduled messages</z-link> for this conversation.
+    {{#*inline "z-link"}}<a href='/#scheduled'>{{> @partial-block}}</a>{{/inline}}
+    {{/tr}}
+    {{else}}
+    {{#tr}}You have <z-link>{scheduled_message_count} scheduled message</z-link> for this conversation.
+    {{#*inline "z-link"}}<a href='/#scheduled'>{{> @partial-block}}</a>{{/inline}}
```

Comment:
I think you can get rid of `several_messages` and instead use ICU plurals -- that allows translators to do plurals correctly in languages that have more complex plural rules (Like in Russian, 11 has a different ending than 10).

https://zulip.readthedocs.io/en/latest/translating/internationalization.html#frontend-translations and the section just after talk about how to do this.